massdriver-application-helm/variables.tf

```diff
@@ -35,3 +35,15 @@ variable "additional_envs" {
   )
   default = []
 }
+
+variable "resource_group_name" {
+  description = "Azure only, the name of resource group to create the Managed Identity in."
+  type        = string
+  default     = null
+}
+
+variable "location" {
```

Comment:
Are there any realistic alternatives to having to pass this? I briefly searched and didn't find anything.

Reply:
I couldn't find any, PRs / ideas welcome. <3

Reply:
(besides getting it from the resource group in golang like we talked about)

Reply:
Issue created: https://github.com/massdriver-cloud/terraform-provider-mdxc/issues/34